python/graphframes/graphframe.py

```diff
@@ -337,12 +335,7 @@ def parallelPersonalizedPageRank(self, resetProbability = 0.15, sourceIds = None
         """
         assert sourceIds is not None and len(sourceIds) > 0, "Source vertices Ids sourceIds must be provided"
         assert maxIter is not None, "Max number of iterations maxIter must be provided"
-        if isinstance(sourceIds, list):
-            _gateway = self._sc._gateway
-            _srcIds = _gateway.new_array(_gateway.jvm.Object, len(sourceIds))
-            for i, vid in enumerate(sourceIds):
-                _srcIds[i] = vid
-            sourceIds = _srcIds
+        sourceIds = self._sc._jvm.PythonUtils.toArray(sourceIds)
```

Comment:
Using the `PythonUtils.toArray` method to convert `Python list` type. Thanks @felixcheung !